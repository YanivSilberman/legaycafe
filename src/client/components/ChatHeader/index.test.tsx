import * as React from 'react';
import * as ReactShallowRenderer from 'react-test-renderer/shallow'
import Component from './index';

/*
selectUsers={[]}
classes={{}}
usersTyping={[]}
subscribeToUserTyping={() => null}
*/

describe('ChatHeader component test',() => {
  it('component loads', () => {
    const renderer = ReactShallowRenderer.createRenderer();

    expect(renderer.render(
      <Component
        chat="chat"
      />
    )).toMatchSnapshot()
  })

  // when typing, shows indicators
});
