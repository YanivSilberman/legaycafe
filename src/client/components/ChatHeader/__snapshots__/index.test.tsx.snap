// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatHeader component test component loads 1`] = `
<Apollo(ChatHeader)
  chat="chat"
  classes={
    Object {
      "appBar": "Apollo(ChatHeader)-appBar-1",
      "avatar": "Apollo(ChatHeader)-avatar-3",
      "menuButton": "Apollo(ChatHeader)-menuButton-5",
      "placeholderAvatar": "Apollo(ChatHeader)-placeholderAvatar-4",
      "title": "Apollo(ChatHeader)-title-6",
      "toolbar": "Apollo(ChatHeader)-toolbar-2",
      "typingIndicator": "Apollo(ChatHeader)-typingIndicator-7",
    }
  }
/>
`;
